org.uqbar.project.wollok.lib/src/wollok/lib.wlk

```diff
@@ -193,7 +230,8 @@ object assert {
 	}
 	
 	/**
-	 * Throws an exception with a custom message. Useful when you reach an unwanted code in a test.
+	 * Throws an exception with a custom message. 
+	 * Useful when you reach an unwanted code in a test.
```

Comment:
I think `code` is uncountable, so I would remove `an`

Reply:
Maybe `unwanted` is not clear, and we could use `you reach code that should not be reached`.